spec/features/04_movies_create_spec.rb

```diff
@@ -0,0 +1,205 @@
+require "rails_helper"
+
+feature "Create Movies" do
+  context "As an authenticated user" do
+    before do
+      user1 = User.create(
+        first_name: "John",
+        last_name: "Doe",
+        username: "johndoe",
+        email: "john@doe.com",
+        password: "abcdef1",
+        newsletter: true
+      )
+      visit "/"
+      click_link("Sign In")
+      fill_in "Email", with: user1.email
+      fill_in "Password", with: user1.password
+      click_button "Sign In"
+    end
+
+    movie1 = Movie.new(
+      title: "Citizen Kane",
+      studio: "Studio",
+      year: 1940,
+      rating: "R",
+      genre: "drama",
+      cast: "Cast Member 1, Cast Member 2",
+      director: "Director 1",
+      screen_writer: "Screen Writer 1"
+    )
+
+    scenario "User sees a link Add new movie" do
+      visit "/"
+      expect(page).to have_content("Add New Movie")
+      expect(page).to have_current_path('/')
+    end
+
+    scenario "User clicks link add new movie, is redirected to"\
+      " new movie form, and sees all the required fields" do
+      visit "/"
+      click_link "Add New Movie"
```

Comment:
there should be a space between the `click_link` and `expect(page)` portions of this test